esp/src/src-react/components/Queries.tsx

```diff
@@ -212,23 +214,63 @@ export const Queries: React.FunctionComponent<QueriesProps> = ({
         { key: "divider_2", itemType: ContextualMenuItemType.Divider, onRender: () => <ShortVerticalDivider /> },
         {
             key: "Suspend", text: nlsHPCC.Suspend, disabled: !uiState.isSuspended,
-            onClick: () => { WsWorkunits.WUQuerysetQueryAction(selection, "Suspend"); }
+            onClick: () => {
+                WsWorkunits.WUQuerysetQueryAction(selection, "Suspend").then(response => {
+                    response.forEach(({ WUQuerySetQueryActionResponse }) => {
+                        WUQuerySetQueryActionResponse.Results.Result.forEach(result => {
+                            selection.filter(s => s.Id === result.QueryId)[0].Suspended = result.Suspended;
+                        });
+                    });
+                    refreshTable();
```

Comment:
If there is a call to refreshTable, then any "selection" syncing should be handled internally by useFluentPagedGrid?